packages/trader/src/Modules/Trading/Helpers/contract-type.js

```diff
@@ -0,0 +1,88 @@
+import { localize } from '@deriv/translations';
+
+export const unsupported_contract_types_list = [
+    // TODO: uncomment before merge
```

Comment:
?

Reply:
@negarn the contracts here is not supported in deriv yet. I comment it out for testing purposes for now. it's a reminder to uncomment it before merging :)